templates/cpp-template-default/Classes/HelloWorldScene.cpp

```diff
@@ -30,9 +37,18 @@ bool HelloWorld::init()
                                            "CloseNormal.png",
                                            "CloseSelected.png",
                                            CC_CALLBACK_1(HelloWorld::menuCloseCallback, this));
-    
-    closeItem->setPosition(Vec2(origin.x + visibleSize.width - closeItem->getContentSize().width/2 ,
-                                origin.y + closeItem->getContentSize().height/2));
+    auto w = closeItem->getContentSize().width;
+    auto h = closeItem->getContentSize().height;
+    if (w <= 0 || h <= 0)
```

Comment:
@TheCodez I agree that this is ugly, however in my test it did return a closeItem, with size 0x0, when the files were not there, so it works as intended.

Could improve it by:
```
if (closeItem == nullptr || 
    closeItem->getContentSize().width <= 0 || 
    closeItem->getContentSize().height <= 0)
```
I'm fairly new to cocos2d-x, is there a better way to check if `MenuItemImage::create` loaded the `.png`s correctly?